src/components/cards/Card.tsx

```diff
@@ -8,13 +16,26 @@ interface ICard {
   icon?: string
   price?: string
 }
-function Card({ border, title, price, icon }: ICard) {
+
+function CoinCard({ title, price, icon }: ICard) {
   return (
-    <div className={`card ${border}`}>
-      <img className=" card-icon" src={icon} alt="icon of the coin" />
-      <h3>{title}</h3>
-      <p>{price} USD</p>
-    </div>
+    <Card sx={{ width: 170 }}>
+      <CardHeader
+        avatar={
+          <Avatar aria-label="recipe">
+            <img className=" card-icon" src={icon} alt="icon of the coin" />
+          </Avatar>
+        }
+      />
+      <CardContent>
+        <Typography variant="body1" color="text.secondary">
+          {title}
+        </Typography>
+        <Typography variant="body2" color="text.secondary">
+          {price}$
```

Comment:
Small remark, but usually dollar sign stands in front of the amount.

Reply:
Also, you could use `FormattedNumber` front `react-intl` to format this based on the currency used. The component accepts a currency prop and style. https://formatjs.io/docs/react-intl/components/#formattednumber
Furthermore, this stops working if the price is calculated in another currency (you are currently hardcoding the dollar sign). Not sure if you get the currency used for the price from the API response, but if so, use that instead of harcoded value.